crazy_miner/telemedicine_views.py

```
@@ -0,0 +1,386 @@
+        # مسیر فایل APK نسبت به همین اپ
+        current_directory = os.path.dirname(__file__)
+        file_path = os.path.join(current_directory, 'apps', 'app-release.apk')
+
+        if not os.path.exists(file_path):
+            return Response({"error": "File not found"}, status=404)
+
+        # ارسال سیگنال شمارش (بدون IP — فقط ثبت شمارش)
+        apk_downloaded.send(sender=self.__class__)
+
+        # پاسخ دانلود فایل
+        response = FileResponse(open(file_path, 'rb'), as_attachment=True, filename='helssa.apk')
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)